scopus-import/src/main/java/no/sikt/nva/scopus/conversion/files/ScopusFileConverter.java

```diff
@@ -167,10 +162,35 @@ private static Collector<ScopusFile, Object, ArrayList<ScopusFile>> collectRemov
                                  ArrayList::new);
     }
 
+    private static Collector<File, Object, ArrayList<AssociatedArtifact>> collectRemovingDuplicatedFiles() {
+        return collectingAndThen(toCollection(() -> new TreeSet<>(comparing(File::getName))), ArrayList::new);
+    }
+
     private static boolean isCreativeCommonsLicense(URI uri) {
         return uri.toString().contains(CREATIVECOMMONS_DOMAIN);
     }
 
+    private static boolean hasSameVersion(CrossrefLink crossrefLink, License license) {
+        return license.getContentVersion().equals(crossrefLink.getContentVersion());
+    }
+
+    private static URI toUri(String string) {
+        return attempt(() -> new URI(string.replace(WHITESPACE, ENCODED_WHITESPACE))).orElseThrow();
+    }
+
+    private static boolean fileWithContent(ScopusFile file) {
+        return file.size() != ZERO_LENGTH_CONTENT;
+    }
+
+    private static boolean isElsvierPlainTextResource(CrossrefLink crossrefLink) {
+        return !(ELSEVIER_HOST.equals(crossrefLink.getUri().getHost())
```

Comment:
Should this be negated? The method is named "isElsevierPlainTextResource()" and with negating it looks like it checks if it is _not_ an ElsevierTextResource

Reply:
my bad, it should be isNot... thanks!